README.md

```diff
@@ -16,8 +16,21 @@ Dummy is a konnector used for development, debugging and test purpose. It is use
 Install
 -------
 
-Since `dummy` is only for development purpose, it must not be available in the Cozy store and as such is only published
-on the `dev` channel.
+Since `dummy` is only for development purposes, it must not be available in the Cozy store and as such is only published
+on GitHub and on the registry in the dev channel. Travis automatically updates thoses branches and the registry when a
+change lands on master.
+
+### Flavors
+
+- `build`: Dummy connector
+- `build_aggregator`: Used to test the aggregator account feature used in Banking konnectors
+- `build_form`
+- `build_oauth`
+- `build_twofa`: Used to test 2FA flow for example for the [Amazon](https://github.com/konnectors/amazon) connector
+
+Each of this flavor is published on a build branch, for example [here](https://github.com/konnectors/dummy/tree/build_twofa)
+
+Additionally, the 2FA flavor is published in the registry at `registry://dummy/dev`.
```

Comment:
```suggestion
Additionally, the 2FA flavor is published in the registry at `registry://dummy/dev`. It can be useful for flows
where it is not suitable to install the connector via the CLI.
```